runtime/plugin/manpager.vim

```diff
@@ -0,0 +1,18 @@
+if empty($MAN_PN)
+  finish
+endif
+
+command! -nargs=0 MANPAGER call s:MANPAGER() | delcommand MANPAGER
+
+fun! s:MANPAGER()
+  bwipe!
+
+  let page = expand('$MAN_PN')
+  let page_pattern = '\v\w+(\-\w+)*'
+  let sec_pattern = '\v\(' . '\w+(\+\w+)*' . '\)'
+  let sec = substitute(matchstr(page, sec_pattern), '\v^\(|\)$', '', 'g')
+  let page = substitute(page, sec_pattern, '', '')
```

Comment:
matchlist instead of matchstr ? `:Man` only handles a single man page as input by `:help Man`.